ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -65,7 +65,7 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 	<xs:simpleType name="paddingType">
 		<xs:restriction base="xs:token">
 			<xs:pattern
-				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+				value="((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%)))(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*"
```

Comment:
Aren't there a whole bunch of `[+-]?` bits missing from this? E.g. in the 2 value version the second one doesn't seem to be allowed a sign.